source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,136 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 - [Release v7.7 - Feature Release](#release-v7-7-feature-release)
 - [Release v7.5 - Feature Release](#release-v7-5-feature-release)
 - [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.8 - [Extended Support Release](https://docs.mattermost.com/upgrade/release-definitions.html#extended-support-release-esr)
+
+**Release Day: February 16, 2023**
+
+### Important Upgrade Notes
+
+ - Before upgrading, we recommend checking for duplicate data in ``focalboard_category_boards`` table and deleting all but one copy of duplicate data. This is to ensure that the new plugin version startup doesn't lock the table, preventing users from using Boards. We recommend de-duplicating the data at a time of low user activity. See more details in the [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
```

Comment:
```suggestion
 - Before upgrading, we recommend checking for duplicate data in the ``focalboard_category_boards`` table, and deleting all but one copy of duplicate data. This is to ensure that the new plugin version startup doesn't lock the table, and prevent users from using Boards. We recommend de-duplicating the data at a time of low user activity. See more details in the [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
```

Reply:
cc @harshilsharma63

Reply:
@amyblais can we use the full release notes mentioned in this PR body - https://github.com/mattermost/focalboard/pull/4409 ? It contains instructions on how to check for duplicate records and another command on how to de-duplicate it.

Reply:
@cwarnermm the table is located with all other mattermost tables. Nothing special.

Reply:
@amyblais @cwarnermm I think the changelog when viewed in preview site is confusing and misleading. If you check this page - http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6187/install/self-managed-changelog.html (which I arrived at by searching for "Changelog" in search bar then opening the result titled "Mattermost self-hosted changelog") you'll see it asks to "See more details in the Important Upgrade Notes" while it is already being said in the Important Upgrade Notes. I thought maybe I missed specying the de-duplication SQL or something but turns out, if you use the sidebar in docs site and navigate from "Upgrade Mattermost" to "Important Upgrade Notes", arriving at this page - http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6187/upgrade/important-upgrade-notes.html there you see all the detailed instruction with corresponding SQLs.

Also, in the sidebar the section about "Important Upgrade Notes" is way after the "Prepare to upgrade Mattermost" and "Upgrade Mattermost Server". I feel "Important Upgrade Notes" should be the first thing in there, before any other upgrade-related steps so that its not a surprise for someone following instructions step by step, top to bottom.

<img width="440" alt="Screenshot 2023-02-06 at 5 58 00 PM" src="https://user-images.githubusercontent.com/18575143/216971326-8dc09823-34bb-4dca-a4ad-fa8fe97479dd.png">

Reply:
Thanks, @harshilsharma63, for this great feedback. I agree that the organization and order of our admin content needs work. This reorg effort is in progress.

Regarding the duplication of content within the self-hosted changelog and the important upgrade notes page, my understanding is that the changelog is comprehensive and includes _all_ changes delivered within a given release, and the important notes are intended to highlight upgrade steps required. @amyblais - have I described that accurately from your perspective?

Reply:
Yes, that's correct.

Reply:
Thanks @cwarnermm @amyblais ! The new changes look great!